youtube_dl/extractor/plays.py

```diff
@@ -9,29 +9,55 @@
 
 class PlaysTVIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?plays\.tv/video/(?P<id>[0-9a-f]{18})'
-    _TEST = {
-        'url': 'http://plays.tv/video/56af17f56c95335490/when-you-outplay-the-azir-wall',
+    _TESTS = [{
+        'url': 'https://plays.tv/video/57698d69e6c84884a2/just-a-typical-reaper-potg',
+        'md5': 'c4bdc35a0c9ca1f95c6ecad2f99c0e76',
+        'info_dict': {
+            'id': '57698d69e6c84884a2',
+            'ext': 'mp4',
+            'title': 'Solfect - Just a typical Reaper POTG',
+            'description': 'Posted by Solfect',
+        }
+    }, {
+        'url': 'https://plays.tv/video/56af17f56c95335490/when-you-outplay-the-azir-wall',
         'md5': 'dfeac1198506652b5257a62762cec7bc',
         'info_dict': {
             'id': '56af17f56c95335490',
             'ext': 'mp4',
-            'title': 'When you outplay the Azir wall',
+            'title': 'Bjergsen - When you outplay the Azir wall',
             'description': 'Posted by Bjergsen',
         }
-    }
+    }, {
+        'url': 'https://plays.tv/video/58236095d388d140aa/-?from=user',
+        'md5': '09535d1d157f679e6446fb39ce51645a',
+        'info_dict': {
+            'id': '58236095d388d140aa',
+            'ext': 'mp4',
+            'title': 'akn3ex - \u63a7\u3048\u3081\u306b\u8a00\u3063\u3066\u9811\u5f35\u3063\u3066\u308b',
+            'description': 'Posted by akn3ex',
+        }
+    }, {
+        'url': 'http://plays.tv/video/56d77f74866aa25854/pro-abortion-propaganda?from=search&search=dota+2',
+        'md5': '3e3fc1d5c20cb373eb3143bbb4ca5070',
+        'info_dict': {
+            'id': '56d77f74866aa25854',
+            'ext': 'mp4',
+            'title': 'SavageIs - pro abortion propaganda',
+            'description': 'Posted by SavageIs',
+        }
+    }]
```

Comment:
Are those test cases different in terms of how videos are extracted? If so please add some comments in each test case. Otherwise just leave one test case.